examples/nodejs/js/src/main.js

```diff
@@ -17,16 +17,14 @@ app.use(bodyParser.json())
 let dvcClient
 
 async function startDVC() {
-    dvcClient = DVC.initialize('<DVC_SERVER_KEY>', { logLevel: 'error' })
-    await dvcClient.onClientInitialized()
-    console.log('DVC onClientInitialized')
+    dvcClient = DVC.initialize('<DVC_SERVER_KEY>', { logLevel: 'info', enabledCloudBucketing: true })
```

Comment:
should be "enableCloudBucketing" instead of "enabledCloudBucketing"

Reply:
good catch! :) fixing